stories/Hamburger/Hamburger.regression-test.story.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+
+import Hamburger from '../../components/Hamburger';
+
+storiesOf('Hamburger', module)
+  .add('default', () => (
+    <span
+      style={{
+        backgroundColor: '#000',
+        display: 'inline-block',
+      }}
+    >
+      <Hamburger />
+    </span>
+  ))
+  .add('showNotification', () => (
+    <span
+      style={{
+        backgroundColor: '#000',
+        display: 'inline-block',
+      }}
+    >
+      <Hamburger showNotification />
+    </span>
+  ))
+  .add('isOpened', () => (
+    <span
```

Comment:
What you think about to create a `styledSpan` component with `styled-components` lib instead to repeat this span with inline style ?

Reply:
done